### PR TITLE
Feature/spk 48

### PR DESCRIPTION
**Pull request by AlexMisiulia**

### CODE REVIEW

File: app/src/main/java/com/simplifire/yotinfcclient/fragment/FormListFragment.kt [hunk 1]
```diff
@@ -93,5 +95,25 @@ class FormListFragment : Fragment() {
         viewModel.error().observeEvent(this) {
             Toast.makeText(activity, R.string.select_form_to_delete_error_msg, Toast.LENGTH_SHORT).show()
         }
+
+        viewModel.descriptionText().observe(this, Observer {
+            if (it != null) {
```
Comment (cquemin) [inline app/src/main/java/com/simplifire/yotinfcclient/fragment/FormListFragment.kt]: No need for null check as the type is `!`

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -67,4 +67,20 @@ number of days or months, when do you want them to expire?</string>
 
     <!--Create form screen end-->
 
+
+    <!--Delete form screen start-->
+
+    <string name="delete_form_list_item_hint">Please tap the name of the form you want to delete.
+        It is often better to leave old forms in the system for future reference</string>
```
Comment (cquemin) [inline app/src/main/res/values/strings.xml]: should this contain a `\n` or not ?

File: app/src/main/java/com/simplifire/yotinfcclient/fragment/FormListFragment.kt (hunk 1 above)
Comment (AlexMisiulia) [reply app/src/main/java/com/simplifire/yotinfcclient/fragment/FormListFragment.kt]: Agree, don't like to call Java classes from Kotlin, that's why the type is nullable

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (AlexMisiulia) [reply app/src/main/res/values/strings.xml]: No. This is just to avoid long lines

**Pull request merged**